packages/auth/src/accessToken/routes.ts

```diff
@@ -64,26 +66,25 @@ async function introspectToken(
   const introspectionResult = await deps.accessTokenService.introspect(
     body['access_token']
   )
-  if (introspectionResult) {
-    ctx.body = introspectionToBody(introspectionResult)
-  } else {
-    ctx.throw(404, {
-      error: 'invalid_request',
-      message: 'token not found'
-    })
-  }
+  ctx.body = introspectionResult
+    ? introspectionToBody(introspectionResult)
+    : {
+        active: false
+      }
 }
 
-function introspectionToBody(result: Introspection) {
-  if (!result.active) return { active: result.active }
-  else {
-    return {
-      active: result.active,
-      grant: result.id,
-      access: result.access?.map((a: Access) => accessToBody(a)),
-      key: result.key,
-      client_id: result.clientId
-    }
+function introspectionToBody({ grant, jwk }: Introspection): ActiveTokenInfo {
+  return {
+    active: true,
+    grant: grant.id,
+    access: grant.access.map((a: Access) =>
+      accessToBody(a)
```

Comment:
Not related to this PR, but `accessToBody()` seems a bit odd to me:
```ts
export function accessToBody(access: Access) {
  return Object.fromEntries(
    Object.entries(access.toJSON()).filter(
      ([k, v]) =>
        v != null &&
        k != 'id' &&
        k != 'grantId' &&
        k != 'createdAt' &&
        k != 'updatedAt'
    )
  )
}
```
I feel like it could be updated to just take the keys that it needs, instead of filtering most of them out?

Reply:
- https://github.com/interledger/rafiki/issues/973

Postponing until after :point_down: is merged:
- https://github.com/interledger/rafiki/pull/956